docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,213 @@
+# Contributing to Devworkspace-Operator
+
+Hello there! thank you for choosing to the contributing to devfile/devworkspace-operator. Navigate through the following to understand more about contributing here.
+
+- [Contributing to Devworkspace-Operator](#contributing-to-devworkspace-operator)
```

Comment:
For naming, we should stick to `DevWorkspace Operator` for the project, or `devworkspace-operator` if referring to kube objects.

```suggestion
- [Contributing to the DevWorkspace Operator](#contributing-to-devworkspace-operator)
```